app/views/exam_templates/_boot.js.erb

```diff
@@ -0,0 +1,5 @@
+<script>
```

Comment:
Do not put `script` tags here. Instead, put it in the view that is rendering this partial.